docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -1,173 +1,173 @@
 rules:
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(([^\\.]+)\\.)?([^\\.]*?).authorization\"><>(\\w+)"
+- pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"BrokerMetrics\", name=\"pinot\\.broker\\.(([^\\.]+)\\.)?([^\\.]*)\\.authorization\"><>(\\w+)"
```

Comment:
Does this follow java regex? If so `[^\\.]` should be equivalent to `[^.]`
Can you share the documentation for the prometheus config? Do all fields take regex as input?

Reply:
Just validated both metrics with `[^\\.]` and `[^.]`, you are right both are equivalent. Will make that update too.
Regarding the documentation for the prometheus config here is a [link](https://github.com/prometheus/jmx_exporter/tree/release-0.20.0?tab=readme-ov-file#configuration) I referred.
According to the doc, only `pattern` field takes a regex while capture groups from pattern can be used in labels.

Reply:
Done.